README.md

```diff
@@ -96,7 +96,7 @@ end
 You can disable headless mode by passing `ferrum` settings:
```

Comment:
I think this `ferrum` should be `driver_options`.

Reply:
Thanks, eagle eye!